python/cuml/cuml/dask/__init__.py

```diff
@@ -12,7 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-import dask
+
+try:
+    import dask
+    import dask.distributed as _  # noqa
+    import dask_cudf as _  # noqa
+    import raft_dask as _  # noqa
+
+    del _
+except ImportError as exc:
```

Comment:
```suggestion
except ModuleNotFoundError as exc:
```

I think that might be better with the same reasoning as outlined [here](https://docs.pytest.org/en/stable/deprecations.html#import-or-skip-import-error).